t/t2210-job-manager-bugs.t

```diff
@@ -60,5 +60,15 @@ test_expect_success 'issue3051: clean up' '
 	flux queue drain
 '
 
+#
+# Issue 3218 job-manager: segfault when changing priority of running job
+#
+
+test_expect_success 'issue3218: cannot reprioritize job with alloc success' '
```

Comment:
apologies for not noticing earlier, but the commit description needs some editing:

> Add test ensuring segfault from attempt to re-prioritize a running job does not lead to a segfault.

Maybe go with

> Add test ensuring job manager does not segfault when a priority change is requested on a running job.

While you're changing it, the commit title should probably not just reference an issue number and the test output could say something a little more descriptive.  Maybe go with
> issue3218: priority change on running job doesn't segfault